pkgs/games/anki/addons/anki-button-colours/default.nix

```diff
@@ -0,0 +1,27 @@
+{
+  lib,
+  anki-utils,
+  fetchFromGitHub,
+  nix-update-script,
+}:
+anki-utils.buildAnkiAddon (finalAttrs: {
+  pname = "anki-button-colours";
+  version = "2.1.1";
+  src = fetchFromGitHub {
+    owner = "teaqu";
+    repo = "anki-button-colours";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-PI2Whtm5/q/X4cnFyANjFQtErOXqdDTSBWPXJjlyTj8=";
+  };
+  passthru.updateScript = nix-update-script { };
+  meta = {
+    description = "Button colours for Anki";
```

Comment:
Maybe something more descriptive like "Change the colours of the answer buttons"?